docs/ops/normalization/NormalizeL2_1.md

```diff
@@ -4,7 +4,22 @@
 
 **Category**: *Normalization*
 
-**Short description**: *NormalizeL2* operation performs L2 normalization of the 1st input tensor in slices specified by the 2nd input.
+**Short description**: *NormalizeL2* operation performs L2 normalization of the first input tensor in slices specified by the second input.
+
+**Detailed Description**
+
+Each element in the output is the result of division of corresponding element from the `data` input tensor by the result of L2 reduction along dimensions specified by the `axes` input:
+
+    output[i0, i1, ..., iN] = x[i0, i1, ..., iN] / sqrt(eps_mode(sum[j0,..., jN](x[j0, ..., jN]**2), eps))
+
+Where indices `i0, ..., iN` run through all valid indices for the `data` input and summation `sum[j0, ..., jN]` have `jk = ik` for those dimensions `k` that are not in the set of indices specified by the `axes` input of the operation.
+`eps_mode` selects how the reduction value and `eps` are combined. It can be `max` or `add` depending on `eps_mode` attribute value.
+
+Particular cases:
+
+1. If `axes` is an empty list, then each input element is divided by itself resulting value `1` for all non-zero elements.
+2. If `axes` contains all dimensions of input `data`, a single L2 reduction value is calculated for entire input tensor and each input element is divided by that value.
+
 
 **Attributes**
 
```

Comment:
I think you can remove this line in \*eps\* attribute: For example, \*eps\* equal to 0.001 means that 0.001 is used if all the values in normalization are equal to zero.

Additionally maybe would be nice to rewrite the description and avoid using added/maximized and to/with

Reply:
Done

Reply:
Done